Agents/PubChemAgent/README.md

```diff
@@ -0,0 +1,194 @@
+# PubChem Agent
+
+## Description
+
+The `PubChem Agent` is a simple API to communicate with The World Avatar (TWA) knowledge graph to query for the requested information and autopopulate the missing data using the PubChem PUG REST API using the OntoSpecies ontology.
```

Comment:
Whilst not wrong, this description is not quite consistent with the general picture we'd like to paint. The agent itself is an agent, not an API. Being an agent, it is fully part of, and does not "communicate with", TWA. A more preferable view is: The agent enables TWA to augment itself upon request with chemical information, which is being queried from PubChem using the API you mention, and represented semantically using OntoSpecies. Please reword.

Reply:
I changed the agent description in the README file as requested